File: Easy/Strings/ReverseString.java
class Solution {
    public String reverseString(String s) {
        char[] charArr = s.toCharArray();
        int i =0;
        int j = charArr.length-1;
        while(i<j)
        {
        	char temp = charArr[i];
        	charArr[i] = charArr[j];
        	charArr[j] = temp;
        	i++;
        	j--;
        }
        return new String(charArr);
    }
}

File: CSharp/Easy/Strings/ReverseString.cs
/*
Write a function that takes a string as input and returns the string reversed.

Example:
Given s = "hello", return "olleh".
*/

public class Solution {
    public string ReverseString(string s) {
        char[] charArr = s.ToCharArray();
        int i=0;
        int j = charArr.Length-1;
        while(i<j)
        {
        	char c = charArr[i];
        	charArr[i] = charArr[j];
        	charArr[j] = c;
        	i++;
        	j--;
        }
        return new string(charArr);
    }
}